docker/README.md

```diff
@@ -34,7 +32,7 @@ docker run --cap-add MKNOD --cap-add SYS_ADMIN --device /dev/fuse \
     -e LOCAL_UID="$(id -u)" -e LOCAL_GID="$(id -g)" \
     -v redox-"$(id -u)-$(id -g)"-cargo:/usr/local/cargo \
     -v redox-"$(id -u)-$(id -g)"-rustup:/usr/local/rustup \
-    -v "$(pwd):$(pwd)" -w "$(pwd)" --rm redox make fetch all
+    -v "$(pwd):$(pwd)" -w "$(pwd)" --rm redoxos/redox make fetch all
 ```
 #### Linux with security modules<br>
```

Comment:
Would you mind removing the `<br>` I left behind? :)

Reply:
Force pushed a new version without it.

Reply:
Thanks